src/main/java/com/cryptobid/backend/model/Auction.java

```diff
@@ -0,0 +1,76 @@
+package com.cryptobid.backend.model;
+
+import com.cryptobid.backend.util.UserType;
+
+import java.sql.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Temporal;
+
+@Entity
+public class Auction {
+
+	@Id
+	@GeneratedValue
+	@Column(nullable = false)
+	private int id;
+
+	@Temporal(TemporalType.TIMESTAMP)
+	@Column(nullable = false)
+	private Date startTime;
+	
+	@Temporal(TemporalType.TIMESTAMP)
+	@Column(nullable = false)
+	private Date endTime;
+
+	@Column(nullable = false)
+	private int initialValue;
+
+	@Column
+	private String status;
```

Comment:
```suggestion
	private AuctionStatus status;
```

Create an enum file for AuctionStatus. (`util/AuctionStatus`)
The auction status are `PENDING`,`ONGOING`,`COMPLETED`,`CANCELLED`